app/views/ingredients/_form.html.erb

```diff
@@ -8,12 +8,17 @@
           <%= f.label :name, class: "form-label" %>
         </div>
         <div class="col-md-8">
-          <%= f.text_field :name, class: "form-control", placeholder: "Enter ingredient", autofocus: true, class: "form-control" %>
+          <%= f.text_field :name, placeholder: "Enter ingredient", autofocus: true, class: "form-control" %>
         </div>
       </div>
       <div class="col mb-3">
         <%= f.submit(@ingredient.new_record? ? "Add Ingredient" : "Update Ingredient",
                                       class: "btn btn-primary btn-lg") %>
+        <% if  logged_in? && (current_chef.admin?) && (@ingredient.new_record? ==false) %> 
```

Comment:
this is minor, but you don't need parentheses around the `current_chef.admin?` check and the `(@ingredient.new_record?` check